### PR TITLE
Feedback Updates

### PR DESCRIPTION
**Pull request by groundwater**
Incorporated some feedback from a few folks.

cc @anjuan @kytrinyx

I can't tag you as reviewers, but I would appreciate your thoughts.

### CODE REVIEW

File: Electron Engineer.md [hunk 1]
```diff
@@ -22,22 +22,26 @@ Our teams goal is to make Electron the best option for building desktop applicat
...
-- you have participated in a desktop software project
-- you have some experience with c/c++
+- you have worked on or with Electron, Node.js, or Chromium
+- you have some experience with compiled languages
 - you have experience collaborating with a global community
 - you think of risk as something to be managed, not avoided
 - you have acted to increase the inclusivity of projects you've worked on 
-- you can list 10 things you love about JavaScript
-- you can demonstrate creative mastery of one technical topic
+- you can talk about why you love about JavaScript
```
Comment (kytrinyx) [inline Electron Engineer.md]: stray word: s/about //

File: Electron Engineer.md [hunk 2]
```diff
@@ -22,22 +22,26 @@ Our teams goal is to make Electron the best option for building desktop applicat
...
   by pulling in the latest [Node.js](https://github.com/nodejs/node), and [Chromium](https://www.chromium.org/Home) releases,
   giving developers access to the latest in web performance and features
 
-## Why you might be a good fit
+## Essentials for Success
 
-These are just examples of what might mean you're a good fit for the team.
-There are likely other examples we haven't listed.
-Share your experience, even if it's not exactly what's listed here.
+- you have one year of experience as a software engineer
```
Comment (kytrinyx) [inline Electron Engineer.md]: What is it specifically about having a year of experience that you want? Which qualities are they?

Comment (groundwater) [inline Electron Engineer.md]: In my experience with people who are either new grads, or from a bootcamp, there is a 6 month period of "new grad" onboarding that is not required by folks who have at least _some_ experience working. I would like to get to the point where we can hire a new grad, but this is definitely not the right role for that.

Comment (kytrinyx) [inline Electron Engineer.md]: That makes sense.

**Pull request merged**